doc/requirements-doc.txt

```diff
@@ -19,6 +19,7 @@ sphinx-click
 sphinx-copybutton
 sphinx-gallery
 sphinx-jsonschema
+sphinx_tabs.tabs
```

Comment:
I think this needs to be `sphinx-tabs` or something? `pip install sphinx-tabs.tabs` gives an error for me.

**ERROR: Could not find a version that satisfies the requirement sphinx_tabs.tabs (from -r requirements-doc.txt (line 22)) (from versions: none)
ERROR: No matching distribution found for sphinx_tabs.tabs (from -r requirements-doc.txt (line 22))**

Reply:
Should be just `sphinx-tabs`. Fixed.